docs/user-guide/admin-cli.md

```diff
@@ -317,22 +317,20 @@ Remove a tenant, optionally all volumes for a tenant can be removed as well.
 
 Sample:
 ```
-/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm tenant1 --remove-volumes True
+/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant rm --name=tenant1 --remove-volumes
```

Comment:
I'm not really sure what is being asked about here. The way we do choice parsing is by comma seperated values with no spaces. It should be feasible to have `--rights all`  since it's a single value. See [here](https://github.com/vmware/docker-volume-vsphere/blob/master/esx_service/cli/vmdkops_admin.py#L90)